core/src/main/java/org/apache/iceberg/rest/responses/LoadTableResponse.java

```diff
@@ -37,6 +42,56 @@
  */
 public class LoadTableResponse implements RESTResponse {
 
+  private static final String METADATA_LOCATION = "metadata-location";
+  private static final String METADATA = "metadata";
+  private static final String CONFIG = "config";
+
+  public static void toJson(LoadTableResponse loadTable, JsonGenerator generator) throws IOException {
+    generator.writeStartObject();
+
+    generator.writeStringField(METADATA_LOCATION, loadTable.metadataLocation);
+
+    generator.writeFieldName(METADATA);
+    TableMetadataParser.toJson(loadTable.metadata, generator);
+
+    generator.writeObjectFieldStart(CONFIG);
+    for (Map.Entry<String, String> keyValue : loadTable.config.entrySet()) {
+      generator.writeStringField(keyValue.getKey(), keyValue.getValue());
```

Comment:
Do we need a null check here? I believe this throws if `keyValue.getValue()` is null.

Reply:
I copied this from the `TableMetadataParser` 🙈  Browsing through the code, I don't see any null checks.

Reply:
I don't think there should be any null values.